docs/openapi/openapi.yml

```diff
@@ -52,8 +52,14 @@ components:
           tokenUrl: https://example.com/oauth/token
           redirectUri: 'http://localhost:5000/oauth2-redirect.html'
           scopes:
-            'read:presentations': Grants permission to read submitted presentations
-            'write:presentations': Grants permission to submit presentations
+            'resolve:did': Grants permission to resolve DIDs
+            'create:vc': Grants permission issue Verifiable Credentials
+            'verify:vc': Grants permission verify Verifiable Credentials
+            'read:vc': Grants permission to get Verifiable Credentials
+            'update:vc': Grants permission to update the status of Verifiable Credentials
```

Comment:
Is this updating the vc or the revocation list?

Reply:
It's updating the revocation list VC. So both, really. 
I added this myself as #124 didn't cover a scope for this (it's the only one I added). I'm definitely open to better naming or granularity. We could call it `revoke:vc`, but distinguishing between VC types (revocation list vs. all others) seems non-trivial?